composer.lock

```diff
@@ -152,6 +199,16 @@
                 "type": "zip",
                 "url": "https://api.github.com/repos/mollie/mollie-api-php/zipball/b6e9848e4893d1d306fba54da3ac6f4a31a40368",
                 "reference": "b6e9848e4893d1d306fba54da3ac6f4a31a40368",
+            "version": "v2.75.0",
+            "source": {
+                "type": "git",
+                "url": "https://github.com/mollie/mollie-api-php.git",
+                "reference": "ac2052855c631b90092768cf6657f6d5b2f5dd9f"
+            },
+            "dist": {
+                "type": "zip",
+                "url": "https://api.github.com/repos/mollie/mollie-api-php/zipball/ac2052855c631b90092768cf6657f6d5b2f5dd9f",
+                "reference": "ac2052855c631b90092768cf6657f6d5b2f5dd9f",
```

Comment:
@mmaymo This is something from another universe.
Invalid JSON.
Sending a PR fixing it ...